letsencrypt/renewer.py

```diff
@@ -166,6 +167,16 @@ def main(config=None, cli_args=sys.argv[1:]):
     # specify a config file on the command line, which, if provided, should
     # take precedence over this one.
     config.merge(configobj.ConfigObj(cli_config.renewer_config_file))
+    # Ensure that all of the needed folders have been created before continuing
+    uid = os.geteuid()
+    if (not os.path.isdir(cli_config.renewal_configs_dir) or
+        not os.path.isdir(cli_config.config_dir)):
+        print "Could not find config directory. Exiting. "
```

Comment:
I figured it would be best to let it continue to [the for loop](https://github.com/letsencrypt/letsencrypt/pull/945/files#diff-88e9d12789ce14158dde8ef58d5f0e16R181) where it will then raise the same exception as before. Would it be better to manually raise the exception?